.github/json_matrices/supported-languages-versions.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "language": "java",
+        "versions": ["11", "17"],
+        "always-run-versions": ["17"]
+    },
+    {
+        "language": "python",
+        "versions": ["3.8", "3.9", "3.10", "3.11", "3.12"],
+        "always-run-versions": ["3.8", "3.12"]
+    },
+    {
+        "language": "node",
+        "full-versions": ["16.x", "17.x", "18.x", "19.x", "20.x"],
```

Comment:
Please align naming for all clients to we can copy paste relevant code without having an error